city_temperature_app/models.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import (
+    Column,
+    DateTime,
+    Float,
+    ForeignKey,
+    Integer,
+    String,
+)
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBCity(Base):
+    __tablename__ = "cities"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(255))
+    additional_info = Column(String(500))
+
+    temperatures = relationship("DBTemperature", back_populates="city")
+
+
+class DBTemperature(Base):

```

Comment:
Rename it to `Temperature`. We don't use `DBCity` to indicate it's a model; in FastAPI, we differentiate using `model.City`, `schemas.City`, etc.